catalog/spatial/registry/registry-federation-admin-service-impl/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -13,20 +13,46 @@
 -->
 <blueprint xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
            xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0"
+           xmlns:cm="http://aries.apache.org/blueprint/xmlns/blueprint-cm/v1.1.0"
            xsi:schemaLocation="http://www.osgi.org/xmlns/blueprint/v1.0.0 http://www.osgi.org/xmlns/blueprint/v1.0.0/blueprint.xsd">
 
-    <bean id="federationAdminService" class="org.codice.ddf.registry.federationadmin.service.impl.FederationAdminServiceImpl" init-method="init">
+    <bean id="federationAdminService"
+          class="org.codice.ddf.registry.federationadmin.service.impl.FederationAdminServiceImpl"
+          init-method="init">
         <property name="catalogFramework" ref="catalogFramework"/>
-        <property name="registryTransformer" ref="inputTransformer"/>
-        <property name="parser" ref="xmlParser"/>
         <property name="filterBuilder" ref="filterBuilder"/>
+        <property name="parser" ref="xmlParser"/>
+        <property name="registryTransformer" ref="inputTransformer"/>
     </bean>
 
-    <service ref="federationAdminService" interface="org.codice.ddf.registry.federationadmin.service.FederationAdminService"/>
+    <cm:property-placeholder persistent-id="Registry_Federation_Admin_Service"
```

Comment:
Any reason we didn't follow our usual naming convention here for `persistent-id`, i.e., something like `org.codice.ddf.registry.federationadmin.service.FederationAdminService`?